app/services/carrierwave_filename.rb

```diff
@@ -1,17 +1,25 @@
 module CarrierwaveFilename
   def for(attachment, underscore: false)
-    file = attachment.file && attachment.file.path.split('/').last
+    file_path = attachment.file && attachment.file.path
```

Comment:
`path` - we're not using a file backend anywhere are? if not, we should be good.